uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -1,14 +1,14 @@
-/* globals angular */
 /* eslint-env amd, no-mixed-operators:false */
```

Comment:
Refer to http://eslint.org/docs/rules/no-mixed-operators. This is to avid pre-commit earnings about mixing operators like "||" and "&&" during comparison. removes it by refactoring the code.